tests/conftest.py

```diff
@@ -90,6 +94,25 @@ def path_coutwildrnp_json():
 
 
 @pytest.fixture(scope='session')
+def path_coutwildrnp_gpkg():
+    """Creates ``coutwildrnp.gpkg`` if it does not exist and returns the absolute
+    file path."""
+    has_gpkg = "GPKG" in fiona.supported_drivers.keys()
+    if not has_gpkg:
+        raise RuntimeError("GDAL has not been compiled with GPKG support")
+    path = os.path.join(data_dir(), 'coutwildrnp.gpkg')
+    if not os.path.exists(path):
+        filename_shp = _COUTWILDRNP_FILES[0]
+        path_shp = os.path.join(data_dir(), filename_shp)
+        with fiona.open(path_shp, "r") as src:
+            meta = copy.deepcopy(src.meta)
```

Comment:
The `meta` property is a new dict and changes to it don't affect the src, we don't need to copy it here.

Reply:
I'll update after the merge.

Reply:
Good to know. I'm not sure where I picked this habit up from.